whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -16065,22 +16065,37 @@
 
     "000": {
       "inherit": "bib",
+      "[5]": {
+        "aboutEntity": "?record",
+        "property": "recordStatus",
+        "tokenMap": "StatusType",
+        "TODO": "Filter all but c & n for auth, fallbackdefault c"
+      },
       "[6]": {
+        "NOTE": "Posttyp - z - Auktoritetspost",
         "fixedDefault": "z"
       },
       "[7]": null,
       "[8]": null,
+      "[9]": {
+        "NOTE": "Teckenuppsättning - a - UCS/Unicode",
```

Comment:
Better to put this note in the inherited column (as it applies there too).

Reply:
Or not inherit bib:000 at all...

Reply:
Removed inheritance and added more explicit notes with reference to bib